toolkit/imageconfigs/postinstallscripts/remove-tdnf-cache.sh

```diff
@@ -0,0 +1,3 @@
+echo removing tdnf cache
```

Comment:
Can you add the copyright notice to the top of this script?
Ex: https://github.com/microsoft/CBL-Mariner/blob/3d3fcf7ecb1300e436d07a59ed78279ac89f5985/.pipelines/prchecks/PackageBuildPRCheck.yml#L1C1-L2C34